src/+otp/Problem.m

```diff
@@ -129,6 +129,8 @@ function validateNewState(obj, newTimeSpan, newY0, newParameters)
                 error('Y0 must be numeric');
             elseif ~(isempty(obj.ExpectedNumVars) || length(newY0) == obj.ExpectedNumVars)
                 error('Expected Y0 to have %d components but has %d', obj.ExpectedNumVars, length(newY0));
+            elseif isempty(newParameters)
+                error('Parameters cannot be empty');
```

Comment:
This is interesting. What if the problem has no parameters? Do we want to force an empty parameter class? I am fine either way.

Reply:
Yes, it would require a class with no properties.  This makes documentation and usage more consistent.